### PR TITLE
Spelling: GitHub, Log out, of the dev

### PR DESCRIPTION
**Pull request by comradekingu**
| Q             | A
| ------------- | ---
| Translation   | yes
| License       | MIT

Not completely sure what "Thought of the developers" alludes to.
Maybe it should be just "Thought of developers" ?

### CODE REVIEW

File: src/Wallabag/CoreBundle/Resources/translations/messages.en.yml [hunk 1]
```diff
@@ -323,7 +323,7 @@ howto:
         go_import: Go to import
         go_developers: Go to developers
         go_howto: Go to howto (this page!)
-        go_logout: Logout
+        go_logout: Log out
```
Comment (tcitworld) [inline src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: If I understand https://whitelightconcepts.com/login-vs-log-in-logout-vs-log-out/ correctly, I think it should stay in one word.

Comment (comradekingu) [reply src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: How is it used in wallabag? You click the button and what happens?
"Login" makes sense for talking about the process of logging in, the portal as it were. Clicking to do something however, is two words. I can't grasp logging out being more complex than clicking the "Log out"-button…?

Comment (Kdecherf) [reply src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: There's no intermediate page when you click on this link, you're directly logged out. 'Log out' seems more accurate here.

Comment (comradekingu) [reply src/Wallabag/CoreBundle/Resources/translations/messages.en.yml]: @tcitworld It is indicative of an action occuring as a result of clicking it, so "Log out" applies.
You got to the logout page (or "Log out"-page) and click "Log out" to perform an action.

**Pull request merged**